books/SPARK_for_the_MISRA_C_Developer/chapters/02_program_consistency.rst

```diff
@@ -48,39 +48,40 @@ enforcing the well-known practice that macro arguments should be parenthesized
 SPARK does not suffer from these problems, as it relies on semantic inclusion
 of context instead of textual inclusion of content, using ``with`` clauses:
 
-.. code-block:: ada
+.. code:: ada
 
-   with Ada.Text_IO;
+    with Ada.Text_IO;
```

Comment:
why this additional space?

Reply:
All source-code examples in our courses start are aligned at 4 spaces. As far as I know, it's needed for the ReST format.